app/src/main/java/org/greenstand/android/TreeTracker/messages/announcementmessage/AnnouncementViewModel.kt

```diff
@@ -40,6 +40,7 @@ class AnnouncementViewModel(
                 currentUrl = announcement.videoLink
             )
             messagesRepo.markMessageAsRead(messageId)
+            messagesRepo.syncMessages()
```

Comment:
This code mainly marks the message as read. There shouldn't be anything to sync with the server yet. Can you explain why we're syncing here?